packages/components/src/Slider/Slider.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Box, system } from '@marigold/system';
+
+type SliderProps = {};
+
+export const Slider = system<SliderProps, 'input'>(
+  ({ variant = 'slider', ...props }) => {
+    return (
+      <Box
+        as="input"
+        type="range"
+        themeSection="form"
+        variant={variant}
+        {...props}
```

Comment:
Move props that should not be overriden below this spread (like the themsection!) :)

Reply:
Ups ... nevermind. Let's just keep it like this for now.